README.md

```diff
@@ -58,6 +58,7 @@ ctx.render();
     globalTranslationX: <Number>,  // Initial x translation of the snappy context (default 0)
     globalTranslationY: <Number>,  // Initial y translation of the snappy context (default 0)
     scaleLineWidth: <Boolean>,     // If true, the snappy context will scale the line width according to scale (default true)
+    scaleDashLine: <Boolean>,      // If true, the snappy context will scale the dash line according to scale (default true)
```

Comment:
// If true, the snappy context will scale the with of the dashes of dashed line according to scale (default true)